### DISCUSSION TITLE
What happened to Elegosoft?

### DISCUSSION THREAD

**Discussion by dmuysers** [modula3/cm3#1133] [General]
All Elegosoft based M3 web pages have suddenly become unreachable without any previous warning. Besides the source code, they also had the most up-to-date language definition, while, on Github, the "docs" folder contains an old version. I browsed all reachable Elegosoft web pages, there is only question of Odoo etc, as if Modula-3 had never existed. Why didn't they transfer their material on Github before getting rid of M3 if they wanted so?

**Comment by dragisha** [comment]
Here. Please retrieve things you see are missing.

**Comment by dmuysers** [comment]
How to compare since "https://modula3.elegosoft.com/cm3" has disappeared?
One can't even get it with the Wayback machine.
I mailed to Elego, but got no answer.
One thing that hit my attention was missing WIDECHAR char- and text-literals.

**Comment by dragisha** [comment]
https://web.archive.org/web/20220814063707/https://modula3.elegosoft.com/cm3/index.html <https://web.archive.org/web/20220814063707/https://modula3.elegosoft.com/cm3/index.html>

Here. I only now see it is you! Nice to hear from our most senior compatriot after all this time!

**Comment by olaf-wagner** [comment]
On 31 Dec 2022, at 15:39, dmuysers wrote:

> All Elegosoft based M3 web pages have suddenly become unreachable 
> without any previous warning. Besides the source code, they also had 
> the most up-to-date language definition, while, on Github, the "docs" 
> folder contains an old version. I browsed all reachable Elegosoft web 
> pages, there is only question of Odoo etc, as if Modula-3 had never 
> existed. Why didn't they transfer their material on Github before 
> getting rid of M3 if they wanted so?

No need to become agitated. We’ll fix it if anything is missing, but 
likely
not tonight. Nothing has been removed intentionally.

Happy new year!

Olaf
-- 
Olaf Wagner -- elego Software Solutions GmbH -- http://www.elegosoft.com
                Gustav-Meyer-Allee 25 / Gebäude 12, 13355 Berlin, 
Germany
Geschäftsführer: Olaf Wagner | Sitz: Berlin
Handelregister: Amtsgericht Charlottenburg HRB 77719 | USt-IdNr: 
DE163214194

**Reply by dmuysers** [reply]
Now that "https://modula3.elegosoft.com/cm3/" is back, I notice that I was mistaken. Even there, the language definition does not mention the WIDECHAR prefix 'W' for character- enf text-literals introduced by cm3.
